File: src/main/java/com/rabbitown/yaquest/util/Logger.java
package com.rabbitown.yaquest.util;

import org.bukkit.Bukkit;

/**
 * YaQuest logger.
 *
 * @author Yoooooory
 */
public class Logger {

    public static String prefix = "§8[§7YaQuest§8] §e";

    /**
     * Log a INFO message.
     *
     * @param message The message to log.
     */
    public static void info(String message) {
        Bukkit.getConsoleSender().sendMessage(prefix + message);
    }

    /**
     * Log a SEVERE message.
     *
     * @param message The message to log.
     */
    public static void severe(String message) {
        Bukkit.getLogger().severe(message);
    }

    /**
     * Log a WARNING message.
     *
     * @param message The message to log.
     */
    public static void warning(String message) {
        Bukkit.getLogger().warning(message);
    }

}

File: src/main/java/com/rabbitown/yaquest/util/Logger.kt
package com.rabbitown.yaquest.util

import com.rabbitown.yalib.module.locale.YLocale
import com.rabbitown.yaquest.YaQuest
import org.bukkit.Bukkit

/**
 * @author Yoooooory
 */
object Logger {

    val prefix = YLocale.getConsoleMessage("prefix")

    fun info(message: String) = Bukkit.getConsoleSender().sendMessage(prefix + message)
    fun severe(message: String?) = YaQuest.instance.logger.severe(message)
    fun warning(message: String?) = YaQuest.instance.logger.warning(message)

}